host_vars/aarnet-queue.usegalaxy.org.au.yml

```diff
@@ -0,0 +1,90 @@
+# Host specific Rabbit settings
+rabbitmq_users:
+  - user: admin
+    password: "{{ rabbitmq_password_admin }}"
+    tags: administrator
+    vhost: /
+  - user: telegraf
+    password: "{{ rabbitmq_password_telegraf }}"
+    tags: administrator
+    vhost: /
+  - user: galaxy
+    password: "{{ rabbitmq_password_galaxy }}"
+    vhost: galaxy
+  # - user: galaxy_mel2
```

Comment:
you can leave these is so that they are set up. None of the pulsars will connect to them anyway so they won't interfere with anything

Reply:
:)